pkg/service/repair/plan.go

```diff
@@ -229,12 +235,11 @@ func (p keyspacePlans) fillSize(ctx context.Context, client *scyllaclient.Client
 	return sizeReport, nil
 }
 
-func (p keyspacePlans) fillOptimize(smallTableThreshold int64) {
+func (p keyspacePlans) fillSmall(smallTableThreshold int64) {
 	for _, ksp := range p {
 		for j, tp := range ksp.Tables {
-			// Return merged ranges for small, fully replicated table (#3128)
-			if tp.Size < smallTableThreshold && tp.ReplicaSetCnt == 1 {
```

Comment:
Is it the answer to my prev quesiton:
> I don't follow. Could you please elaborate why len(ring.ReplicaTokens) == 1 ?
Isn't tp.Small just enough ?
?
So, the logic is moved from here ?
But this is the number of replicasets,.

Reply:
`tp.ReplicaSetCnt` and `len(ring.ReplicaTokens)` should be equal. I tried to answer the question in the reply to previous comment.

Reply:
Right, I found the logic in plan.go and the comment
```
// ReplicaTokenRanges describes all token ranges belonging to given replica set.
```

```
			for _, rep := range ring.ReplicaTokens {
				filtered := filterReplicaSet(rep.ReplicaSet, ring.HostDC, target)
				if len(filtered) == 0 {
					continue
				}

				replicaSetCnt++
				allHosts.Add(filtered...)

				for _, h := range filtered {
					ranges[newHostKsTable(h, u.Keyspace, t)] += len(rep.Ranges)
				}
				rangesCnt += len(rep.Ranges)
			}
```

Looks that ReplicaToken name is bit misleading, could be ReplicaTokensPerReplicaSet.